packages/runtime-core/src/helpers/useModel.ts

```diff
@@ -51,8 +51,9 @@ export function useModel(
       },
 
       set(value) {
+        const _localValue = options.get ? options.get(localValue) : localValue
         if (
-          !hasChanged(value, localValue) &&
+          !hasChanged(value, _localValue) &&
```

Comment:
```suggestion
          !hasChanged(value, options.get ? options.get(localValue) : localValue) &&
```

Reply:
thanks for your suggestion,and tests had been added